binding/Binding/VkTypes.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace SkiaSharp
+{
+	public struct VkHandle
+	{
+		public IntPtr Handle;
```

Comment:
Is there a reason for this as a separate struct as opposed to just an `IntPtr`? Also, are we using this type?

Reply:
IntPtr doesn't give any information on the semantics of the pointer, would rather have a type. It's currently unused because I didn't bind anything that required a vulkan handle (e.g. instance, device, allocators). Just seemed weird to define the non dispatchable handle and not the handle.